.github/ISSUE_TEMPLATE/01_bug.yml

```
@@ -0,0 +1,81 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
```

```suggestion
labels: ["Bug"]
```
The label's first letter is captial on this repo, not on forks.